test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -429,24 +535,82 @@ define([
                 });
 
                 it('can start in an internal error state', async function () {
+                    setUpCellInState(this, 'INTERNAL_ERROR');
+                    await startRunningCell(this);
+
+                    expect(this.appCellWidgetInstance.__fsm().getCurrentState().state).toEqual(
+                        fsmState.INTERNAL_ERROR
+                    );
+                    // the reset button should be active
+                    const resetButton = this.kbaseNode.querySelector(selectors.reset);
+                    expect(resetButton).not.toHaveClass('hidden');
+                    expect(resetButton).not.toHaveClass('disabled');
+                });
+
+                it('shows an internal error if the FSM throws a hissy fit', async function () {
                     this.cell.metadata.kbase.appCell.fsm = {
-                        currentState: { mode: 'internal-error' },
+                        currentState: { this: 'does', not: 'exist' },
                     };
-                    await this.appCellWidgetInstance.init();
-                    await this.appCellWidgetInstance.attach(this.kbaseNode);
-                    await this.appCellWidgetInstance.start();
-                    await this.appCellWidgetInstance.run();
+                    await startRunningCell(this);
 
                     expect(this.appCellWidgetInstance.__fsm().getCurrentState().state).toEqual(
                         fsmState.INTERNAL_ERROR
                     );
-                    // the
+                    const expectedError = {
+                        title: 'Error initializing app state',
+                        message: 'Cannot find initial state',
+                        advice: [
+                            'Reset the app and try again.',
+                            'If that fails, delete the app cell and create a new one.',
+                        ],
+                        info: null,
+                        detail: null,
+                    };
+                    expect(this.appCellWidgetInstance.model.getItem('internalError')).toEqual(
+                        expectedError
+                    );
+
+                    [...expectedError.advice, expectedError.title, expectedError.message].forEach(
+                        (text) => {
+                            expect(this.kbaseNode.textContent).toMatch(text);
+                        }
+                    );
```

Comment:
Rather than exhaustively testing the layout of the error pane, just make sure that the error text shows up.